### PR TITLE
FORUM-1240 The topic Title is cut even there is a blank space

### PR DESCRIPTION
**Pull request by trongtt**

### CODE REVIEW

File: platform-ui-skin/src/main/webapp/skin/less/forum/portlets/forum/forum.less [hunk 1]
```diff
@@ -83,7 +87,10 @@
 }
 
 @media(max-width: 955px) {
-
+		.uiForumPortlet .breadcrumb .active {
```
Comment (thomasdelhomenie) [inline platform-ui-skin/src/main/webapp/skin/less/forum/portlets/forum/forum.less]: Indentation is not consistent.

File: platform-ui-skin/src/main/webapp/skin/less/forum/portlets/forum/forum.less [hunk 2]
```diff
@@ -102,7 +109,7 @@
         display: block;
     }
 }
-@media(max-width: 768px) {
+@media(max-width: 768px) {
```
Comment (thomasdelhomenie) [inline platform-ui-skin/src/main/webapp/skin/less/forum/portlets/forum/forum.less]: Useless change

File: platform-ui-skin/src/main/webapp/skin/less/forum/portlets/forum/forum.less (hunk 1 above)
Comment (trongtt) [inline platform-ui-skin/src/main/webapp/skin/less/forum/portlets/forum/forum.less]: Fixed.

However, if you look at the file you will see it is not consistent itself in overall :)

Comment (thomasdelhomenie) [inline platform-ui-skin/src/main/webapp/skin/less/forum/portlets/forum/forum.less]: Yes you are right, but let's try to not increase the inconsistency ;)

**Comment by thomasdelhomenie** [comment]
LGTM

**Comment by trongtt** [comment]
merged

**Pull request closed without merging**